### PR TITLE
lightning: better estimate of the max-open-files

### PR DESCRIPTION
**Pull request by glorv**
### What problem does this PR solve? 
Make better estimate of the max-open-files by also take import routines into account.

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Release note

 - No release note

### CODE REVIEW

File: pkg/lightning/backend/local/local_unix.go [hunk 1]
```diff
@@ -39,8 +39,8 @@ func GetSystemRLimit() (Rlim_t, error) {
 // In Local-backend, we need to read and write a lot of L0 SST files, so we need
 // to check system max open files limit.
 func VerifyRLimit(estimateMaxFiles Rlim_t) error {
-	if estimateMaxFiles < minRLimit {
-		estimateMaxFiles = minRLimit
+	if estimateMaxFiles > maxRLimit {
+		estimateMaxFiles = maxRLimit
```
Comment (kennytm) [inline pkg/lightning/backend/local/local_unix.go:43]: the "1024" introduced in pingcap/tidb-lightning#501 is no longer useful?

Comment (glorv) [reply pkg/lightning/backend/local/local_unix.go:43]: Yes, `minRLimit` is to avoid the minlimit calculated too small, but it seems to be not very useful

**Review by kennytm — APPROVED** [review]

**Comment by kennytm** [comment]
merged into pingcap/tidb#27597 already.

**Pull request merged**